src/main/java/trackitnus/model/util/SampleDataUtil.java

```diff
@@ -59,31 +60,31 @@
     public static Lesson[] getSampleLessons() {
         return new Lesson[]{
             new Lesson(new Code("CS1101S"), Type.LEC, new LessonDateTime(DayOfWeek.Fri, LocalTime.NOON,
-                LocalTime.NOON.plusHours(2))),
+                LocalTime.NOON.plusHours(2)), new Address("E-learning")),
             new Lesson(new Code("CS1101S"), Type.TUT, new LessonDateTime(DayOfWeek.Mon, LocalTime.NOON,
-                LocalTime.NOON.plusHours(2))),
+                LocalTime.NOON.plusHours(2)), new Address("E-learning")),
             new Lesson(new Code("CS2100"), Type.LEC, new LessonDateTime(DayOfWeek.Tue, LocalTime.NOON.plusHours(4),
-                LocalTime.NOON.plusHours(6))),
+                LocalTime.NOON.plusHours(6)), new Address("E-learning")),
             new Lesson(new Code("CS2100"), Type.LAB, new LessonDateTime(DayOfWeek.Mon, LocalTime.MIDNIGHT.plusHours(9),
-                LocalTime.MIDNIGHT.plusHours(10))),
+                LocalTime.MIDNIGHT.plusHours(10)), new Address("E-learning")),
             new Lesson(new Code("CS2103T"), Type.LEC, new LessonDateTime(DayOfWeek.Wed, LocalTime.NOON.plusHours(2),
-                LocalTime.NOON.plusHours(4))),
+                LocalTime.NOON.plusHours(4)), new Address("E-learning")),
             new Lesson(new Code("CS2103T"), Type.TUT, new LessonDateTime(DayOfWeek.Thu, LocalTime.NOON.plusHours(2),
-                LocalTime.NOON.plusHours(3))),
+                LocalTime.NOON.plusHours(3)), new Address("E-learning")),
             new Lesson(new Code("MA1101R"), Type.LEC, new LessonDateTime(DayOfWeek.Thu, LocalTime.NOON.plusHours(4),
-                LocalTime.NOON.plusHours(6))),
+                LocalTime.NOON.plusHours(6)), new Address("E-learning")),
             new Lesson(new Code("MA1101R"), Type.SEC, new LessonDateTime(DayOfWeek.Wed, LocalTime.NOON,
-                LocalTime.NOON.plusHours(1))),
+                LocalTime.NOON.plusHours(1)), new Address("E-learning")),
             new Lesson(new Code("GER1000H"), Type.LEC, new LessonDateTime(DayOfWeek.Tue, LocalTime.NOON,
```

Comment:
Hi, I think the address should be more realistic (throw in some sample zoom links, some venues at the school like COM1-0120....)

Reply:
Thanks for the review! Currently, the app doesn't accept strings with '/' as valid parameters, hence it is impossible to type in links. Should we add support for links in the future? I'm still leaving some of the test cases as "E-learning" as placeholder.

Reply:
hmm it's quite a big problem. Anyway, can you change the address to some random locations in NUS to make it more realistic? Because this is the sample data we will show the user too!